### ISSUE TITLE
User.Voice.Read return the amount you want to read.

### ISSUE THREAD

**Issue by JordanMotta** [martindevans/MumbleSharp#66]
I am not sure if this is a bug or not. But I was searching inside `AudioDecodingBuffer.cs and` and inside method `Read` I saw that in case you have no data to read the method returns 0, but I all the time I get the same amount that I want to read. So I have no way to know whether the data was read succefully or not. I tried to place a big buffer (96000) and sometimes I get a different values (normally smaller) but other times I get the same size of the buffer.

Basically it never returns 0. Is this the wanted behavior?
